docs/sphinx/source/whatsnew/v0.6.0.rst

```diff
@@ -14,6 +14,24 @@ Enhancements
 * Add sea surface albedo in irradiance.py (:issue:`458`)
 * Implement first_solar_spectral_loss in modelchain.py (:issue:`359`)
 * Clarify arguments Egref and dEgdT for calcparams_desoto (:issue:`462`)
+* Implement a reliable "gold" implementation of the single diode model (SDM)
```

Comment:
These additions to whatsnew seem out of date.
- need to mention `singlediode_methods` and what new functions are attached, and what functions (if any) have moved from `pvsystem`
- would help to list the values for the new kwarg `method` on `singlediode` and what they do
- I think `bishop88` is in `singlediode_methods` not `pvsystem` , the reference needs to be corrected elsewhere.

Reply:
I exposed the two new main functions, `bishop88` and `estimate_voc` in `pvsystem`, at the top of the module just after the imports - my thought process being that the methods in `singlediode_methods` would be too low level for most folks, but not necessarily private.

I'll go with the flow, but perhaps that additional documentation and API change can be added in another PR, perhaps the one where we move the Lambert-W methods to that module?

I'll just add some notes to what's new, that I added this module and put some low level plumbing in there, okay?

Reply:
Agreed. Let's batch move to `singlediode_methods` in a separate PR